hardhat.config.js

```diff
@@ -0,0 +1,164 @@
+        params: [],
+      },
+      {
+        name: 'Add tokens as whitelisted collaterals',
+        id: 'ADD_COLLATERAL_TOKEN_ROLE',
+        params: [],
+      },
+      {
+        name: 'Remove tokens as whitelisted collaterals',
+        id: 'REMOVE_COLLATERAL_TOKEN_ROLE',
+        params: [],
+      },
+      {
+        name: 'Update collateral tokens collateralization settings',
+        id: 'UPDATE_COLLATERAL_TOKEN_ROLE',
+        params: [],
+      },
+      {
+        name: 'Open trading',
+        id: 'OPEN_TRADING_ROLE',
+        params: [],
+      },
+      {
+        name: 'Make buy orders',
+        id: 'MAKE_BUY_ORDER_ROLE',
+        params: ['Buyer'],
+      },
+      {
+        name: 'Make sell orders',
+        id: 'MAKE_SELL_ORDER_ROLE',
+        params: ['Seller'],
+      },
+    ],
+    appBuildOutputPath: 'public/',
+  },
+  networks: {
+    hardhat: {
+      // process.env.HARDHAT_FORK will specify the network that the fork is made from.
+      // this line ensure the use of the corresponding accounts
+      accounts: accounts(process.env.HARDHAT_FORK),
+      forking: process.env.HARDHAT_FORK
+        ? {
+            url: node_url(process.env.HARDHAT_FORK),
+            blockNumber: process.env.HARDHAT_FORK_NUMBER
+              ? parseInt(process.env.HARDHAT_FORK_NUMBER)
+              : undefined,
+          }
+        : undefined,
+    },
+    localhost: {
+      url: node_url('localhost'),
+      accounts: accounts(),
+      ensRegistry: '0xaafca6b0c89521752e559650206d7c925fd0e530',
+    },
+    mainnet: {
+      url: node_url('mainnet'),
+      accounts: accounts('mainnet'),
+      appEnsName: 'augmented-bonding-curve.aragonpm.eth',
+    },
+    rinkeby: {
+      url: node_url('rinkeby'),
+      accounts: accounts('rinkeby'),
+      ensRegistry: '0x98Df287B6C145399Aaa709692c8D308357bC085D',
+    },
+    xdai: {
+      url: node_url('xdai'),
+      accounts: accounts('xdai'),
+      ensRegistry: '0xaafca6b0c89521752e559650206d7c925fd0e530',
+    },
+    polygon: {
+      url: node_url('polygon'),
+      accounts: accounts('polygon'),
+      ensRegistry: '0x7EdE100965B1E870d726cD480dD41F2af1Ca0130',
+    },
+    mumbai: {
+      url: node_url('mumbai'),
+      accounts: accounts('mumbai'),
+      ensRegistry: '0xB1576a9bE5EC445368740161174f3Dd1034fF8be',
+    },
+    arbitrum: {
+      url: node_url('arbitrum'),
+      accounts: accounts('arbitrum'),
+      ensRegistry: '0xB1576a9bE5EC445368740161174f3Dd1034fF8be',
+    },
+    arbtest: {
+      url: node_url('arbtest'),
+      accounts: accounts('arbtest'),
+      ensRegistry: '0x73ddD4B38982aB515daCf43289B41706f9A39199',
+    },
+    frame: {
+      url: 'http://localhost:1248',
+      httpHeaders: { origin: 'hardhat' },
+      timeout: 0,
+      gas: 0,
+    },
+  },
+  ipfs: {
+    gateway: 'https://ipfs.blossom.software/',
+    pinata: {
+      key: process.env.PINATA_KEY || '',
+      secret: process.env.PINATA_SECRET_KEY || '',
+    },
+  },
```

Comment:
`ipfs` config extension gives us control over the IPFS connection that `hardhat-aragon` setup for us. Check the [README of the plugin](https://github.com/1Hive/hardhat-aragon/blob/master/README.md) for more details.